src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -1052,7 +1051,7 @@ public enum LegendsAwakened implements LogicCardInfo {
           pokeBody "Ditto DNA", {
             text "As long as Ditto is your Active Pokémon, its maximum HP is the same as your opponent's Active Pokémon. Ditto can use the attacks of that Pokémon as its own. (You still need the necessary Energy to use each attack.) If that Pokémon is no longer your opponent's Active Pokémon, choose 1 of your opponent's Active Pokémon for Ditto to copy."
             getterA (GET_FULL_HP, self) { Holder h->
-              if (self.active) h.object = hp(self.owner.opposite.pbg.active.getFullHP())
+              if (self.active) h.object = hp(self.owner.opposite.pbg.active.topPokemonCard.getFullHP())
```

Comment:
`getFullHP` does not exist on the PokemonCard class. It only exists on the `PokemonCardSet` class.